session/src/main/java/org/apache/iotdb/session/pool/SessionPool.java

```diff
@@ -292,6 +357,34 @@ public void insertTablets(Map<String, Tablet> tablets)
     insertTablets(tablets, false);
   }
 
+  /**
+   * use batch interface to insert data in an asynchronous manner
+   * @param tablets
+   * @param timeout asynchronous call timeout in millisecond
+   */
+  public CompletableFuture<Integer> asyncInsertTablets(Map<String, Tablet> tablets, boolean sorted,
```

Comment:
This method has a same one in session, maybe we can abstract a class named AsyncSession which contains in both Session and SessionPool